music21/musicxml/m21ToXml.py

```diff
@@ -4774,6 +4828,15 @@ def expressionToXml(self, expression):
             if k in classes:
                 mx = Element(v)
                 break
+        if mx is None:
+            # 'ArpeggioMark' maps to two different elements
+            if 'ArpeggioMark' in classes:
```

Comment:
See Myke's posts about v8 on the list, we're moving to more standard `isinstance()` instead of .classes

Reply:
Agreed.  I'll leave the older 'if k in classes' alone, though (for now).